### PR TITLE
feat: add possibility to delete a expense line from a draft expense

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#72430

### CODE REVIEW

**Review by vhu-axelor — COMMENTED** [review]
Work well, nice! 🤩

File: packages/apps/hr/src/features/expenseLineSlice.js [hunk 1]
```diff
@@ -72,21 +73,25 @@ export const searchGeneralExpenseLines = createAsyncThunk(
 
 export const deleteExpenseLine = createAsyncThunk(
   'expenseLine/deleteExpenseLine',
-  async function (data = {}, {getState}) {
+  async function (data = {}, {getState, dispatch}) {
     return handlerApiCall({
       fetchFunction: _deleteExpenseLine,
       data,
       action: 'Hr_SliceAction_DeleteExpenseLine',
```
[the remark this answers is not in the document]
Comment (gca-axelor) [reply packages/apps/hr/src/features/expenseLineSlice.js:80]: like what ?

**Review by vhu-axelor — APPROVED** [review]
Nice!

File: packages/apps/hr/src/components/molecules/ExpenseLineDetailCard/ExpenseLineDetailCard.tsx [hunk 2]
```diff
@@ -150,7 +154,8 @@ const ExpenseLineDetailCard = ({
                 onPress={onEdit}
                 style={styles.cardIconButton}
               />
-              {expense == null && (
+              {(expense == null ||
+                expense.statusSelect === Expense.statusSelect.Draft) && (
```
Comment (lme-axelor) [inline packages/apps/hr/src/components/molecules/ExpenseLineDetailCard/ExpenseLineDetailCard.tsx:158]: That's the same condition as above or I'm missing something ? 🤔

**Pull request merged**